tests/core/mountpoint/test_folder_operations.py

```diff
@@ -213,7 +213,11 @@ def move_folder(self, src, dst_parent, dst_name):
         def iterdir(self, path):
             expected_exc = None
             try:
-                expected_children = {x.name for x in path.to_oracle().iterdir()}
+                expected_children = {
+                    x.name
+                    for x in path.to_oracle().iterdir()
+                    if not x.name.__contains__(".fuse_hidden")
```

Comment:
`".fuse_hidden" in x.name` is the pythonic way to do this ^^

but In the current case I would use `x.name.startswith(".fuse_hidden")` instead